src/main/java/frc/robot/subsystems/OldClimber.java

```diff
@@ -55,57 +55,30 @@ public OldClimber() {
 
         rightWinchMotor.setNeutralMode(NeutralMode.Brake);
         leftWinchMotor.setNeutralMode(NeutralMode.Brake);
-        //negative percent output values bring climber in, positive bring it out.
 
-        toggleLock();
-    }
-
-    public void toggleLock() {
-        // switch (getLockState()) {
-        //     case Locked:
-        //         setLockState(RungLockState.Unlocked);
-        //         break;
-        //     case Unlocked:
-        //         setLockState(RungLockState.Locked);
-        //         break;
-
-        // }
-    }
-
-    public void setLockState(RungLockState lockState) {
-        // rungLockSolenoid.set(lockState.getLockStateValue());
-    }
-
-    public RungLockState getLockState() {
-    //     switch (rungLockSolenoid.get()) {
-    //         case kForward:
-    //         case kOff:
-    //             return RungLockState.Unlocked;
-    //         case kReverse:
-    //         default:
-    //             return RungLockState.Locked;
-
-    //     }
-        return RungLockState.Unlocked;
-    }
-
-    public boolean isLockFullyUnlocked() {
-        // TODO sensor things. return bool if sensors say
-
-        return false;
-    }
-
-    public boolean isLockFullyLocked() {
-        // TODO sensor things. return bool if sensors say
-
-        return false;
+        rightWinchMotor.config_kD(0, CLIMBER_MOTOR_KD);
+        rightWinchMotor.config_kP(0, CLIMBER_MOTOR_KP);
+        rightWinchMotor.configAllowableClosedloopError(0, CLIMBER_MOTOR_ALLOWABLE_ERROR);
+        leftWinchMotor.config_kD(0, CLIMBER_MOTOR_KD);
+        leftWinchMotor.config_kP(0, CLIMBER_MOTOR_KP);
+        leftWinchMotor.configAllowableClosedloopError(0, CLIMBER_MOTOR_ALLOWABLE_ERROR);
+        
+        resetEncoders();
+        // is this an alternative to magnetic encoders?
+        // rightWinchMotor.configForwardSoftLimitThreshold(ENCODER_TICKS_TO_ARMS_LENGTH_DIVIDED_BY_ONE);
+        // leftWinchMotor.configForwardSoftLimitThreshold(ENCODER_TICKS_TO_ARMS_LENGTH_DIVIDED_BY_ONE);
     }
 
     public void setArmDown() {
         armSolenoid.set(Value.kForward);
 
     }
 
+    public void setArmPostion (double armLength) {
+        rightWinchMotor.set(ControlMode.Position, armLength*ARM_LENGTHS_TO_ENCODER_TICKS);
+        leftWinchMotor.set(ControlMode.Position, armLength*ARM_LENGTHS_TO_ENCODER_TICKS);
+        // well too bad because Augie's at the keyboard so have a random comment that makes no sense if you weren't in the programming room on 3/26/2022 at 11:40AM.
```

Comment:
```suggestion
```
@KnightPiscesAugie I really enjoy your hidden stories, and lets please keep them as local machine easter eggs instead of in the public repo